src/sentry/api/endpoints/project_details.py

```diff
@@ -220,6 +221,7 @@ def put(self, request, project):
                         'detail': ['The new team is not found.']
                     }, status=400
                 )
+            old_team = project.team
```

Comment:
Can we just use `project.team_id` instead of fully resolving `project.team` ? It seems not necessary since you can just use the team id in the query below.